internal/session/repository_connection.go

```diff
@@ -416,8 +416,8 @@ func (r *ConnectionRepository) closeOrphanedConnections(ctx context.Context, wor
 		ctx,
 		db.StdRetryCnt,
 		db.ExpBackoff{},
-		func(reader db.Reader, w db.Writer) error {
-			rows, err := r.reader.Query(ctx, fmt.Sprintf(orphanedConnectionsCte, notInClause), args)
```

Comment:
This seems like something we could write a [ruleguard](https://github.com/quasilyte/go-ruleguard) linter for 🤔. Great find!

Reply:
A check for unused variables would also catch it. I know some linters provide that, but I don't think we have it enabled.